Retention Policies licensing required for teams #2904 (#3020)
MicrosoftDocs/OfficeDocs-SkypeForBusiness: Teams/retention-policies-faq.md

By default, Teams chat, channel, and files data are retained forever. A user can delete something, but in the absence of retention policies, Teams data is always archived into Exchange online mailboxes (user and group) and stays there for eDiscovery. 

> ### What is the licensing required for retention policy? 

Reviewer: ```suggestion
### What is the licensing requirement for a retention policy? 
```

> Usually, all those that contain Microsoft Teams and an active Exchange mailbox:

Reviewer: ```suggestion
Usually, all those licenses that contain Microsoft Teams and an active Exchange mailbox:
```

- Office 365 Business Premium
- Office 365 Business Essentials

> - Office 365 E1, E3 or E5

Reviewer: ```suggestion
- Office 365 E1, E3, or E5
```
